src/Reflection/ReflectionParameter.php

```diff
@@ -459,6 +477,48 @@ public function isDefaultValueConstant(): bool
         return $this->isDefaultValueConstant;
     }
 
+    /**
+     * Checks parameter type.
```

Comment:
```suggestion
     * Checks if a parameter has the type specified in `$type`
```
or something similar? Just need something more descriptive